src/renderer/components/settings/group.vue

```diff
@@ -1,7 +1,8 @@
 <template>
-    <div>
-        <h4 v-if="group.name">{{ group.name }}</h4>
+    <div class="ui segment">
+        <h4 class="ui dividing header" v-if="group.name">{{ group.name }}</h4>
```

Comment:
Why is it `dividing`?

Reply:
No specific reason, it just seems like a nice touch to how it's look

Reply:
Try to keep the page clean of unnecessary elements. But if you think it helps the flow of the page then keep it.

Reply:
Yes, because I don't want the header to be big on the one side and I want something to separate between the header and the fields.

Reply:
A simple header also has a margin, which a dividing header does not have. But again, if you feel it helps the flow of the page, leave it as is.